optuna/pruners/hyperband.py

```diff
@@ -240,21 +199,14 @@ def _try_initialization(self, study: "optuna.study.Study") -> None:
             self._total_trial_allocation_budget += trial_allocation_budget
             self._trial_allocation_budgets.append(trial_allocation_budget)
 
-            if self._min_early_stopping_rate_low is None:
-                min_early_stopping_rate = bracket_id
-            else:
-                min_early_stopping_rate = self._min_early_stopping_rate_low + bracket_id
-
             _logger.debug(
-                "{}th bracket has minimum early stopping rate of {}".format(
-                    bracket_id, min_early_stopping_rate
-                )
+                "{}th bracket has minimum early stopping rate of {}".format(bracket_id, bracket_id)
```

Comment:
Not sure this log is any useful any longer and wondering if we shouldn't just remove it.

Reply:
Thanks for the comment. That makes sense. I will remove it.